### ISSUE TITLE
meteor.logInStatus() got an error

### ISSUE THREAD

**Issue by YouSour** [tanutapi/dart_meteor#63]
hi @tanutapi, on my backend when user A change password, all devices that user A logged in must be logout too, so i try meteor.logInStatus() to check it but i got an error below, any idea ? thank in advance.
```
Unhandled Exception: Invalid argument(s) (onError): The error handler of Future.catchError must return a value of the future's type
```
example code : 
```
StreamBuilder<UserLogInStatus>(
        stream: meteor.logInStatus(),
        builder: (context, logInStatus) {
          print(logInStatus.data);
          print(logInStatus.data == UserLogInStatus.loggedIn);
          return const Login();
        },
      ),
```

**Comment by YouSour** [comment]
solved

**Comment by tanutapi** [comment]
Could you share your solution? Thank you.On 27 Jan 2023, at 15:27, Sour ***@***.***> wrote:﻿
Closed #63 as completed.

**Comment by YouSour** [comment]
@tanutapi just a temporary solution, uncheck uncaught exceptions on debug  but error still exist, don't know why i think u should check it. 
```
Unhandled Exception: Invalid argument(s) (onError): The error handler of Future.catchError must return a value of the future's type
```
